manage.py

```diff
@@ -93,7 +101,7 @@ def seed(include_interventions=False, keep_unmentioned=False):
     SitePersistence().import_(include_interventions, keep_unmentioned)
 
 
-@manager.command
+@app.cli.command()
 def export_site():
```

Comment:
@pbugni I noticed `export_site()` writes to the same file we read from (set by `PERSISTENCE_FILE`). Would it make more sense to write to a file specified by an argument instead?